lib/src/SortableList/SortableList.tsx

```diff
@@ -162,6 +163,57 @@ export const SortableList: React.FC<SortableListProps> = ({ onSort, className, d
         [dragContainerRef, draggingOrders, dragNode, setIsDragging, setCurrentItemIndex, setCurrentItemNode, onSort, setDefaultOrders]
     );
 
+    const onItemSwap = React.useCallback(
+        (currentIndex: number, swapIndex: number, focusTarget: HTMLDivElement) => {
+            const newList: OrderItem[] = defaultOrders.slice(0);
+            const swapItem = newList[swapIndex];
+            newList[swapIndex] = newList[currentIndex]; // swap overlapped
+            newList[currentIndex] = swapItem;
+            onSort(newList.map(({ uniqueKey }: OrderItem) => uniqueKey));
+            setDefaultOrders(newList);
+            focusTarget.focus();
+        },
+        [defaultOrders]
+    );
+
+    const onKeyDown = React.useCallback(
+        (event: React.KeyboardEvent<HTMLDivElement>) => {
+            const selectedItemIndex: number = Number((event.target as HTMLDivElement).dataset?.index);
+            switch (event.key) {
+                case Key.Space:
+                case Key.Enter:
+                    event.preventDefault();
```

Comment:
is this needed? this seemed to be preventing the checkbox toggle behaviour within the sortable list

Reply:
tested, and you are right xD removed